activesupport/lib/active_support/current_attributes.rb

```diff
@@ -164,6 +164,7 @@ def method_missing(name, *args, &block)
 
           send(name, *args, &block)
         end
+        ruby2_keywords(:method_missing)
```

Comment:
```suggestion
        ruby2_keywords(:method_missing) if respond_to?(:ruby2_keywords, true)
```

Otherwise it fails like on CI:
```
undefined method `ruby2_keywords' for #<Class:ActiveSupport::CurrentAttributes>
```

In 7.0 we don't need it because we know `ruby2_keywords` is defined.

Reply:
Thanks, I just noticed that in CI. Fixed and rebased.